### DISCUSSION TITLE
I want to ask about css framework

### DISCUSSION THREAD

**Discussion by Nutrijelly** [pure-css/pure#1355] [General]
Sorry in advance, let me introduce myself, my name is Kevin, a 12th grade vocational high school student. I really like making websites based on pure HTML and CSS, sorry without javascript because I haven't studied it yet.. okay let's continue, sometimes I make my own version of a website page or follow someone else's design then I build html and css according to the website I see, especially the results are responsive. The point is I can build the foundation of a website and decorate it, so I wonder if in the world of work can use pure css or does it depend on the background of the industry itself? and if you need a framework, which css framework is worth trying, should I try to learn them all? thank you that's my question :)

**Comment by redonkulus** [comment]
Hello and welcome! I'm glad to see you are interested in learning web development. It's always changing, and the library recommendations you may get today could be outdated a year from now. However, one thing that never gets out of date is learning the basics.

For HTML, that's [semantic markup](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) and [accessibility](https://www.w3schools.com/html/html_accessibility.asp). For CSS, learning the basics of CSS without a library is important. Fundamentals like cascading styles and [responsive design](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design) are core to getting comfortable with CSS. 

Once you have tried these concepts in your projects, you can start researching popular CSS frameworks like Tailwind to understand why websites migrate to them. As for this library, Pure CSS tries to normalize some things and create some simple components to build your website.
